src/persist/src/runtime.rs

```diff
@@ -102,7 +103,7 @@ where
 
     // Start up the ticker thread.
     let ticker_tx = tx.clone();
-    let ticker_handle = async_runtime.spawn(async move {
+    let ticker_handle = async_runtime.spawn_named(|| "persist_ticker", async move {
```

Comment:
our thread names seem to have inherited a `:` hierarchy (e.g. "timely:work-<num>") from timely that's mostly consistent across the codebase. maybe we should use the same for async tasks? (to be clear, I don't care and 100% defer to @benesch's opinion on this)

Reply:
The rest of this pr uses `<module_thing:extra_context>` but we can play around with the naming scheme as we get more experience

eventually context will be in the structured task data as opposed to the name and we can improve the names from there

Reply:
I have not formed an opinion! Very happy to let someone else be the one to develop one.